pyproject.toml

```diff
@@ -83,6 +83,12 @@ cz_conventional_commits = "commitizen.cz.conventional_commits:ConventionalCommit
 cz_jira = "commitizen.cz.jira:JiraSmartCz"
 cz_customize = "commitizen.cz.customize:CustomizeCommitsCz"
 
+[tool.poetry.plugins."commitizen.format"]
```

Comment:
Should it be changelog_format or something similar to make it more readable?

Reply:
It's a free form string so yes, it can `changelog_provider` but I chose `format` only for the follozing reason:
- I try as much as possible to stick with single word namings
- It was consistent with other endpoints: `commitizen.provider`, `commitizen.scheme` and `commitizen.plugin` which should be more like `commitizen.version_provider`, `commitizen.version_scheme`, `commitizen.core_plugin` based on the same reasonning
- It was consistent with the package name too `commitizen.format`
- it has been a very long iteration and the name comes from a past discussion in this PR (can't remind which one)

But except those points, I don't have a strong opinion on this, so I can rename it if you prefer

Reply:
If there were already discussions on it, I'm good with it. @woile what do you think?

Reply:
I like `changelog_provider`, and I think the others should be changed 🤔 of course no need now. Just `format` can be too open to interpretation

Reply:
Renamed from `commitizen.format` to `commitizen.changelog_format` in the last commit 👍